index.html

```diff
@@ -66,98 +74,101 @@ <h2>Multi-Post Stories</h2>
+				<h2>Profesional Art <br> Printing Data</h2>
 				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
 				   <ul class="lang">
 					<li><a href="">html</a></li>
 					<li><a href="">bootstrap</a></li>
 					<li><a href="">Ruby</a></li>
 				</ul>
-				<button class="btn2">See Project</button>
+				<button class="btn2" type="button">See Project</button>
 			</div>
 		</div>
 
-		<div class="container">
+		<div class="container box2">
 			<div class="project2">
-				<h2>Profesional Art Printing Data</h2>
+				<h2>Profesional Art <br> Printing Data</h2>
 				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
 				   <ul class="lang">
 					<li><a href="">html</a></li>
 					<li><a href="">bootstrap</a></li>
 					<li><a href="">Ruby</a></li>
 				</ul>
-				<button class="btn2">See Project</button>
+				<button class="btn2" type="button">See Project</button>
 			</div>
 		</div>
 
 		<div class="container">
 			<div class="project2">
-				<h2>Profesional Art Printing Data</h2>
+				<h2>Profesional Art <br> Printing Data</h2>
 				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
 				   <ul class="lang">
 					<li><a href="">html</a></li>
 					<li><a href="">bootstrap</a></li>
 					<li><a href="">Ruby</a></li>
 				</ul>
-				<button class="btn2">See Project</button>
+				<button class="btn2" type="button">See Project</button>
 			</div>
 		</div>
 
-		<div class="container">
+		<div class="container box1">
 			<div class="project2">
-				<h2>Profesional Art Printing Data</h2>
+				<h2>Profesional Art <br> Printing Data</h2>
 				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
 				   <ul class="lang">
 					<li><a href="">html</a></li>
 					<li><a href="">bootstrap</a></li>
 					<li><a href="">Ruby</a></li>
 				</ul>
-				<button class="btn2">See Project</button>
+				<button class="btn2" type="button">See Project</button>
 			</div>
 		</div>
 
-		<div class="container">
+		<div class="container box2">
 			<div class="project2">
-				<h2>Profesional Art Printing Data</h2>
+				<h2>Profesional Art <br> Printing Data</h2>
 				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
 				   <ul class="lang">
 					<li><a href="">html</a></li>
 					<li><a href="">bootstrap</a></li>
 					<li><a href="">Ruby</a></li>
 				</ul>
-				<button class="btn2">See Project</button>
+				<button class="btn2" type="button">See Project</button>
 			</div>
 		</div>
+	</div>
 	</section>
 
 	<section class="about-me">
 		<div class="container2">
 			<h2>About Me</h2>
 			<p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and 
```

Comment:
- [OPTIONAL] KINDLY CONSIDER GIVING THIS H2 A PADDING-TOP TO HAVE A VERY GOOD COPY OF THE PROVIDED FIGMA DESIGN
- your design 👇 
![ssss](https://user-images.githubusercontent.com/60146030/221943340-b9f3e8e6-838b-487d-8449-c6500cb5c5a0.PNG)
- Figma design 👇 
![kr](https://user-images.githubusercontent.com/60146030/221943666-58db9382-5fd2-40d5-84cd-0c31312ec58a.PNG)